docs/topics/install-eap-plugin.md

```diff
@@ -24,3 +24,10 @@ You can follow these instructions to install [the preview version of the Kotlin
 4. Click **Install**. 
 
 If you want to work on existing projects that were created before installing the EAP version, you need to [configure your build for EAP](configure-build-for-eap.md). 
+
+## If you run into any problems
+
+* Report issues to [our issue tracker, YouTrack](https://kotl.in/issue).
+* Find help in the _#eap_ channel on [Slack](https://kotlinlang.slack.com/) ([get an invite](https://surveys.jetbrains.com/s3/kotlin-slack-sign-up)).
```

Comment:
It's better to format this text in the same style, like in `Getting Started with Kotlin` (getting-started.md):
* #eap as a link (and not an italic)
* Slack as a plain text (not a link)
The link to the channel is more important than to the Slack

Reply:
I synchronized the formatting with other EAP-related pages. Do I need to update the formatting there too?

Reply:
Yeah,
You can make this whole sentence as a link: [#eap channel in Kotlin Slack]